src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -169,6 +169,26 @@
     </StringProperty.DataSource>
   </StringProperty>
 
+  <StringProperty Name="TargetPlatformIdentifier"
+                  Visible="False">
+    <StringProperty.DataSource>
```

Comment:
You can remove the DataSource, and just mark the property as `ReadOnly="true"`. I know the other properties did this, but they are wrong and I'll fix them.

We don't need the data source; it was copied from ConfigurationGeneral and was only used when writing to the property which we don't ever do from this rule.

Reply:
I fixed the data sources here: https://github.com/dotnet/project-system/pull/6450, so just pull that change and copy the other properties.